### ISSUE TITLE
[bug] "transparent" option stopped working

### ISSUE THREAD

**Issue by MegaSa1nt** [tauri-apps/tauri#6868]
### Describe the bug

![image](https://user-images.githubusercontent.com/49565695/236392658-b3867b9c-854b-4f43-b21b-c72adf5cc546.png)
After i updated tauri to 1.3 it stopped being transparent
This is my window settings:
```json
"windows": [
  {
    "fullscreen": false,
    "resizable": false,
    "title": "GreenCatsServer!",
    "width": 720,
    "height": 680,
	"fileDropEnabled": false,
	"center": true,
	"theme": "Dark",
	"decorations": false,
	"transparent": true
  }
]
```

How it should be:
![image](https://user-images.githubusercontent.com/49565695/236392798-579acf2d-1a8c-4100-978f-b10eede9e578.png)

### Reproduction

1. Update Tauri to 1.3
2. Make transparent background

### Expected behavior

It should be transparent

### Platform and versions

```text
[✔] Environment
    - OS: Windows 10.0.22621 X64
    ✔ WebView2: 112.0.1722.68
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.69.0 (84c898d65 2023-04-16)
    ✔ Cargo: 1.69.0 (6e9a83356 2023-04-12)
    ✔ rustup: 1.25.2 (17db695f1 2023-02-01)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 18.16.0
    - npm: 9.5.1

[-] Packages
    - tauri [RUST]: 1.3.0
    - tauri-build [RUST]: 1.3.0
    - wry [RUST]: 0.24.1
    - tao [RUST]: 0.16.0
    - @tauri-apps/api [NPM]: not installed!
    - @tauri-apps/cli [NPM]: 1.3.0

[-] App
    - build-type: bundle
    - CSP: unset
    - distDir: ../src
    - devPath: ../src
```

**Comment by MegaSa1nt** [comment]
Here is fix: https://discord.com/channels/616186924390023171/1103484113115152424/1103595329175162890
TLDR: add this to Cargo.toml:
```
[patch.crates-io]
tao = { git = "https://github.com/tauri-apps/tao", branch = "v0.16" }
```
